File: test/Automation_python_CodeGen_Quality_RealRepo_python/pipelines/Python_E2E_Existing_Pipeline_All_2/code/python_e2e_existing_pipeline_all_2/graph/SQLStatement_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from python_e2e_existing_pipeline_all_2.config.ConfigStore import *
from python_e2e_existing_pipeline_all_2.udfs.UDFs import *

def SQLStatement_1(spark: SparkSession, in0: DataFrame) -> (DataFrame):

    try:
        registerUDFs(spark)
    except NameError:
        print("registerUDFs not working")

    in0.createOrReplaceTempView("in0")
    df1 = spark.sql("select * from in0")

    return df1

File: test/Automation_scala_CodeGen_Quality_RealRepo_scala/pipelines/Scala_E2E_Existing_Pipeline_All_1/code/src/main/scala/io/prophecy/pipelines/scala_e2e_existing_pipeline_all_1/graph/SQLStatement_1.scala
package io.prophecy.pipelines.scala_e2e_existing_pipeline_all_1.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scala_e2e_existing_pipeline_all_1.udfs.PipelineInitCode._
import io.prophecy.pipelines.scala_e2e_existing_pipeline_all_1.udfs.UDFs._
import io.prophecy.pipelines.scala_e2e_existing_pipeline_all_1.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object SQLStatement_1 {

  def apply(context: Context, in0: DataFrame): (DataFrame, DataFrame) = {
    val spark = context.spark
    in0.createOrReplaceTempView("in0")
    (spark.sql("select * from in0 where C_BIRTH_YEAR!=22"),
     spark.sql("select * from in0 where C_CUSTOMER_SK!=22")
    )
  }

}
